app/decorators/decidim/initiatives/initiatives_controller_decorator.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+# This decorator changes the value of the key :state in default_filter_params Hash.
+Decidim::Initiatives::InitiativesController.class_eval do
```

Comment:
Comment in the README please that this decorator can be removed after `SIGNATURE_START_DATE`.